EIPS/eip-BN256_HashToCurve.md

```diff
@@ -0,0 +1,342 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 3068
```
Don't forget to rename this file to `EIPS/eip-3068.md` as well.

Reply:
I renamed the file and updated it to EIP-3068.